apps/crn-server/src/data-providers/groups.data-provider.ts

```diff
@@ -69,6 +69,10 @@ export class GroupSquidexDataProvider implements GroupDataProvider {
       if (options.filter.userId) {
         filterList.push({ 'data/leaders/iv/user': options.filter.userId });
       }
+
+      if ('active' in options.filter) {
```

Comment:
I could not use

```suggestion
      if (options.filter.active) {
```

because active can come as `false` then it won't enter this condition